docs/modules/analytics/partials/rest-admin/definitions.adoc

```diff
@@ -0,0 +1,291 @@
+
+// This file is created automatically by Swagger2Markup.
```

Comment:
is porting to Redocly on roadmap, out of interest?

Reply:
Actually come to think of it, we may not need to generate static AsciiDoc output from the OpenAPI specs alongside the Redocly output. It may be possible to grab the stuff we need from the OpenAPI spec directly, and use the templates extension and the markdown macro to generate the static output. 🤔